TASagentTwitchBot.Core/Core/Cryptography.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TASagentTwitchBot.Core.Core
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Security.Cryptography;
```

Comment:
I keep the using statements at the top of the file, but also half of these are duplicated from the using statements already there